pkg/utils/package.go

```diff
@@ -2,12 +2,56 @@ package utils
 
 import (
 	"fmt"
+	"github.com/mandelsoft/filepath/pkg/filepath"
+	"github.com/mandelsoft/goutils/general"
+	"github.com/mandelsoft/vfs/pkg/osfs"
+	"github.com/mandelsoft/vfs/pkg/vfs"
+	"golang.org/x/mod/modfile"
 	"reflect"
 	"runtime"
 	"strings"
 )
 
-const MODULE_PATH = "github.com/open-component-model/ocm"
+func GetRelativePathToProjectRoot(i ...int) (string, error) {
```

Comment:
may I ask for golang-doc for that function, please?
What's the optional parameter 'i' for?

Reply:
added documentation